NashTeamAlpha/BangazonUserAuth
# Replaced Active Customer instance with Identity Framework logic

## Pull Request Description

Added user identity using Identity Framework 

## Steps To Test

1. `git prune`
1. `git checkout ddw-user-identity`
1. Make sure that you can login, add a product to your cart, and see it in the shopping cart.
